content/blog/2017/2017-02-02-jenkins-community-survey

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "The State of Jenkins - 2016 Community Survey "
+tags:
+- community
+- survey
+- continuous delivery
+author: brao
+---
+
+[NOTE]
+====
+This is a guest post by Bhavani Rao, Marketing Manager at link:http://cloudbees.com[CloudBees] 
+====
+
+Last fall, prior to Jenkins World, CloudBees conducted a link:https://jenkins.io/blog/2016/09/09/take-the-2016-jenkins-survey-blog/[Community Survey on behalf of the Jenkins community]. We received over 1200 responses  – and thanks to this input, we 
+have some interesting insights into what Jenkins users are doing.
+
+Based on the results, Jenkins is increasingly being used to support continuous delivery (CD). Jenkins 2, which introduced Pipeline-as-code and the Pipeline plugin as core to Jenkins, has skyrocketed to being adopted by nearly half Jenkins users.  Other data remained consistent with findings year to year. For example, the number of Jenkins users continues to increase, and 90% of survey respondents still consider Jenkins mission-critical. 
+
+ link:https://www.cloudbees.com/sites/default/files/2016-jenkins-community-survey.pdf[Get the survey infographic pdf]    |    link:https://www.cloudbees.com/sites/default/files/2016-jenkins-community-survey-responses.pdf[Get the complete survey results in pdf ]
+
+Here are some of the key findings:
+
+The overwhelming majority of respondents, 85% indicated that Jenkins usage had increased. Diving a little deeper, for organizations with more than 50 software projects, almost 30% used Jenkins in 2016 as compared to 16% in 2012
```

Comment:
Not formatted as a list. If unintentional, use `- ` for unordered and `. ` for ordered list.